pylops/transforms/__init__.py

```diff
@@ -0,0 +1,16 @@
+"""
+Transforms
```

Comment:
I am not in favour of making a transform module as we already have other transforms similar to DCT in the signalprocessing module.. whilst this may have been a good idea in the past, to avoid breaking changes we cannot move the other operators and so it would be strange to have a transform module with some transforms not in it…

Reply:
moved the dct operator to signalprocessing in [4a5f759](https://github.com/PyLops/pylops/pull/480/commits/4a5f7594daeada1e383e73c03a2689d576f90eea)